releasenotes/notes/make_user_user_jmx_command_instead_of_check-e374f41fc5cdcddd.yaml

```diff
@@ -0,0 +1,4 @@
+fixes:
+  - |
+    The parameter ``jmx`` is not supported with the command ``check``, it is replaced by the command ``jmx``.
```

Comment:
super minor nit: since we're note really replacing anything this should probably say:
```
The parameter ``jmx`` is not supported with the command ``check``, the ``jmx`` command should be used instead.
```